livecheck/utils/portage_test.py

```diff
@@ -1,12 +1,11 @@
-"""Configuration for Pytest."""
 import pytest
 from livecheck.utils.portage import sanitize_version
 
 
-def test_sanitize_version() -> None:
-    # Test cases for normalize_version function
-    # fmt: off
-    test_cases = [
+# fmt: off
```

Comment:
```suggestion
```

I do not mind if the formatter handles these lines. Also do these comments not break the decorator?